recipes/mastercard/create_mastercard.sql

```diff
@@ -0,0 +1,346 @@
+INSERT INTO mastercard.zip_to_borough values ('11691','QN',4);
+INSERT INTO mastercard.zip_to_borough values ('11096','QN',4);
+INSERT INTO mastercard.zip_to_borough values ('11692','QN',4);
+INSERT INTO mastercard.zip_to_borough values ('11694','QN',4);
+INSERT INTO mastercard.zip_to_borough values ('11697','QN',4);
+INSERT INTO mastercard.zip_to_borough values ('11372','QN',4);
+INSERT INTO mastercard.zip_to_borough values ('11004','QN',4);
+INSERT INTO mastercard.zip_to_borough values ('11040','QN',4);
+INSERT INTO mastercard.zip_to_borough values ('11424','QN',4);
+INSERT INTO mastercard.zip_to_borough values ('11426','QN',4);
+INSERT INTO mastercard.zip_to_borough values ('11365','QN',4);
+INSERT INTO mastercard.zip_to_borough values ('11001','QN',4);
+INSERT INTO mastercard.zip_to_borough values ('11375','QN',4);
+INSERT INTO mastercard.zip_to_borough values ('11427','QN',4);
+INSERT INTO mastercard.zip_to_borough values ('11374','QN',4);
+INSERT INTO mastercard.zip_to_borough values ('11366','QN',4);
+INSERT INTO mastercard.zip_to_borough values ('11423','QN',4);
+INSERT INTO mastercard.zip_to_borough values ('11428','QN',4);
+INSERT INTO mastercard.zip_to_borough values ('11432','QN',4);
+INSERT INTO mastercard.zip_to_borough values ('11379','QN',4);
+INSERT INTO mastercard.zip_to_borough values ('11429','QN',4);
+INSERT INTO mastercard.zip_to_borough values ('11435','QN',4);
+INSERT INTO mastercard.zip_to_borough values ('11415','QN',4);
+INSERT INTO mastercard.zip_to_borough values ('11418','QN',4);
+INSERT INTO mastercard.zip_to_borough values ('11433','QN',4);
+INSERT INTO mastercard.zip_to_borough values ('11451','QN',4);
+INSERT INTO mastercard.zip_to_borough values ('11421','QN',4);
+INSERT INTO mastercard.zip_to_borough values ('11419','QN',4);
+INSERT INTO mastercard.zip_to_borough values ('11434','QN',4);
+INSERT INTO mastercard.zip_to_borough values ('11416','QN',4);
+INSERT INTO mastercard.zip_to_borough values ('11373','QN',4);
+INSERT INTO mastercard.zip_to_borough values ('11371','QN',4);
+INSERT INTO mastercard.zip_to_borough values ('11361','QN',4);
+INSERT INTO mastercard.zip_to_borough values ('11414','QN',4);
+INSERT INTO mastercard.zip_to_borough values ('10301','SI',5);
+INSERT INTO mastercard.zip_to_borough values ('10303','SI',5);
+INSERT INTO mastercard.zip_to_borough values ('10302','SI',5);
+INSERT INTO mastercard.zip_to_borough values ('10304','SI',5);
+INSERT INTO mastercard.zip_to_borough values ('10314','SI',5);
+INSERT INTO mastercard.zip_to_borough values ('10305','SI',5);
+INSERT INTO mastercard.zip_to_borough values ('10306','SI',5);
+INSERT INTO mastercard.zip_to_borough values ('10308','SI',5);
+INSERT INTO mastercard.zip_to_borough values ('10312','SI',5);
+INSERT INTO mastercard.zip_to_borough values ('10309','SI',5);
+INSERT INTO mastercard.zip_to_borough values ('10307','SI',5);
+INSERT INTO mastercard.zip_to_borough values ('10310','SI',5);
```

Comment:
We'll need to double check this list once Mastercard sends us correct SI data.

Reply:
We should implement a check:
- to make sure that the number of rows in the raw and processed data are the same
- if the number of rows is not the same, then most likely there are new zip codes (unlikely, but possible) that we'll need to add to this code (unless they are not NYC).
- what happens if some of this zip codes are dropped?

Reply:
How do you want it to fail if some NYC zip codes are not included in the dataset @objornsson